pages/proposals.jsx

```diff
@@ -211,121 +11,11 @@ var Proposals = React.createClass({
-            <InputCombo onClick={this.themeCheckboxClicked} className="checkbox-input theme-checkbox" for="leadershipTheme" type="checkbox">
-              Leadership
-            </InputCombo>
-            <InputCombo onClick={this.themeCheckboxClicked} className="checkbox-input theme-checkbox" for="openPracticesTheme" type="checkbox">
-              Open Practices
-            </InputCombo>
-            <InputCombo onClick={this.themeCheckboxClicked} className="checkbox-input theme-checkbox" for="placeTheme" type="checkbox">
-              Place
-            </InputCombo>
-            <InputCombo onClick={this.themeCheckboxClicked} className="checkbox-input theme-checkbox" for="privacyTheme" type="checkbox">
-              Privacy
-            </InputCombo>
-            <InputCombo onClick={this.themeCheckboxClicked} className="checkbox-input theme-checkbox" for="scienceTheme" type="checkbox">
-              Science
-            </InputCombo>
-            <InputCombo onClick={this.themeCheckboxClicked} className="checkbox-input theme-checkbox" for="sustainabilityTheme" type="checkbox">
-              Sustainability
-            </InputCombo>
-            <InputCombo onClick={this.themeCheckboxClicked} className="checkbox-input theme-checkbox" for="userControlTheme" type="checkbox">
-              User Control
-            </InputCombo>
-            <InputCombo onClick={this.themeCheckboxClicked} className="checkbox-input theme-checkbox" for="webLiteracyTheme" type="checkbox">
-              Web Literacy
-            </InputCombo>
-            <InputCombo onClick={this.themeCheckboxClicked} className="checkbox-input theme-checkbox" for="youthTheme" type="checkbox">
-              Youth
-            </InputCombo>
-            <InputCombo onClick={this.otherCheckboxClicked} className="checkbox-input theme-checkbox" for="otherTheme" type="checkbox">
-              Other: <input onClick={this.otherInputClicked} className="other-theme-input" type="text"/>
-            </InputCombo>
-            <div id="theme-other-error-message" className="error-message">Other cannot be empty.</div>
-            <div id="theme-error-message" className="error-message">At least one theme is required.</div>
-
-            <InputCombo errorMessage="You must agree to our privacy policy." className="checkbox-input" for="privacyPolicy" type="checkbox">
-              I&lsquo;m okay with Mozilla handling my info as explained in this <a href="https://www.mozilla.org/en-US/privacy/">Privacy Policy</a>.
-            </InputCombo>
-            <button className="button" id="submit-button" ref="submitButton" onClick={this.onSubmit}>Submit <Icon spin name="spinner"/></button>
-            <div id="generic-error" className="error-message">Uh oh! There's an unknown error with your session proposal. Please try again later, or email <a href="mailto:festival@mozilla.org ">festival@mozilla.org</a> for help.</div>
+            <h1>Thanks for your interest!</h1> 
```

Comment:
This doesn't feel like enough? Shouldn't we mention that proposal deadline has passed?